config/initializers/inflections.rb

```diff
@@ -13,14 +13,4 @@
 # end
 
 ActiveSupport::Inflector.inflections(:en) do |inflect|
-  inflect.uncountable %w[
-    expenses_additional_sources
-    household_health_situations
-    household_personal_details
-    household_situations
-    income_additional_sources
-    income_other_assets
-    maybe_submit_documents
-    preferences_reminders
-  ]
```

Comment:
we aren't using these anymore! Were deleted two weeks ago. :)

Back story on why I even found these: I initially forgot that we had a non-AR class named `Address` so when I added the addresses table and tried to make a AR class it wasn't working and I was baffled. One theory I had was that somehow the inflection rules for the app weren't letting Rails see `Address` as the AR class for the `addresses` table...but then I realized we had two classes :)

Can remove in a separate PR, if you like!